src/app/app.component.html

```diff
@@ -1,4 +1,4 @@
-<h1>
+<h1 xmlns="http://www.w3.org/1999/html">
```

Comment:
Could you remove this attribute, please?

Reply:
Done! Wow no idea how that got in there. Good catch! 🙌